requirements-dev.txt

```diff
@@ -2,7 +2,9 @@
 -r requirements.txt
 
 # dev
-black==19.3b0
-flake8-quotes==2.0.1
-flake8==3.7.7
+black==19.10b0
+flake8-quotes==2.1.1
+flake8==3.7.9
+hypothesis==5.1.5
 isort==4.3.21
+pytest-django==3.7.0
```

Comment:
I've opened https://github.com/scaife-viewer/explorehomer-atlas/pull/17 so we can run the tests as part of CI